### ISSUE TITLE
Input var validation sniff flags vars even when value isn't being assigned

### ISSUE THREAD

**Issue by JDGrimes** [WordPress-Coding-Standards/WordPress-Coding-Standards#186]
Say you have this code:

```php
if ( isset( $_POST['some_var'] ) && '-1' === $_POST['some_var'] ) {
     // do stuff
}
```

The second reference to the variable will be flagged as not sanitized, even though its value is not being assigned to another variable, or passed to a function.

**Comment by fbm-static** [comment]
:+1:
